src/ucp/wireup/select.c

```diff
@@ -88,6 +100,38 @@ static const char *ucp_wireup_iface_flags[] = {
     [ucs_ilog2(UCT_IFACE_FLAG_TAG_RNDV_ZCOPY)]   = "tag rndv zcopy"
 };
 
+static const char *ucp_wireup_iface_amo32_op[] = {
+    [UCT_ATOMIC_OP_ADD]                         = "32-bit atomic add",
```

Comment:
seems a lot of code duplication, can we build the status string fro pieces like "32" + "-bit atomic " + "add"?

Reply:
wrapped to macro